README.md

```diff
@@ -0,0 +1,32 @@
+## TDR Jenkinslib
+
+This is a library of groovy functions for the TDR Jenkins set up.
```

Comment:
I would reword this section slightly: 

"This is a library of groovy functions for TDR Jenkins builds to use

As TDR is a multi-faceted project dealing with many Jenkins builds, a shared library of functions limits the amount of repetitive code. It allows changes code in the library to be propogated across all Jenkins jobs that use that function rather than having to change individual functions in each Jenkins job."